src/assets/stylesheets/profile.scss

```diff
@@ -30,13 +44,59 @@
     }
   }
 
+  :local(.tabs) {
+    display: flex;
+    width: 100%;
+
+    a {
+      padding: 8px;
+      text-align: center;
+      flex: 1;
+      &.selected {
+        font-weight: bold;
+        text-decoration: underline;
+      }
+    }
+  }
+
+  :local(.file-input-row) {
+    label {
+      display: flex;
+      align-items: center;
+      margin: 5px 0;
+    }
+    input {
+      display: none;
+    }
+    .img-box {
+      width: 50px;
+      height: 50px;
+      border-radius: 8px;
+      margin-right: 5px;
+      background: #eaeaea;
+      overflow: hidden;
+    }
+    img {
+      width: 100%;
+    }
+  }
+
   :local(.avatar-selector-container) {
     flex: 1;
     position: relative;
     width: 95%;
     text-align: center;
     margin: 16px;
 
+    &.avatar-editor {
```

Comment:
should use `:local` for these nested ones too, its what I've been doing at least :)